src/Options.php

```diff
@@ -356,6 +366,8 @@ public function set($attributes, $value = null)
                 $this->setPdfBackend($value);
             } elseif ($key === 'pdflibLicense' || $key === 'pdflib_license') {
                 $this->setPdflibLicense($value);
+            } elseif ($key === 'setHttpContext' || $key === 'set_http_context' || $key === 'http_context') {
```

Comment:
Should be `$key === 'httpContext' || $key === 'http_context'` for consistency.

Reply:
Fixed